src/main/java/org/openelisglobal/person/valueholder/Person.java

```diff
@@ -62,7 +69,8 @@ public class Person extends BaseObject<String> {
   private String fax;
   @Email private String email;
 
-  private Set patients = new HashSet(0);
+  @AssociationInverseSide(inversePath = @ObjectPath(@PropertyValue(propertyName = "person")))
+  private Set<Patient> patients = new HashSet<>(0);
```

Comment:
Can you move this change ie `Set<Patient> patients  `  to this [PR ](https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/1083)  so that you can add some few Tests on `person.getPatients() `